setup.py

```diff
@@ -19,7 +19,7 @@ def read(fname):
 
 setup(
     name='pygemony',
-    install_requires=read('requirements.txt'),
+    install_requires=read('requirements/all.txt'),
```

Comment:
People who install this from pip will no longer be required to install nose, mock, and wheel even though they don't need them to run this tool.